package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "osrm",
-  "version": "5.11.0",
+  "version": "5.11.0-latest.1",
   "private": false,
```

Comment:
Yeah bumping after release to `{MAJOR}.{MINOR+1}.0-latest.1` should be the best-practice.